products/centreon.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Centreon
+category: server-app
+tags: php-runtime
+permalink: /centreon
+versionCommand: cat /etc/centreon/centreon.conf
```

Comment:
It looks odd to look at a configuration file to obtain a version, are you sure?

Reply:
Thsi one is a sure one, let me check with some guys 💭

Reply:
- https://github.com/centreon/centreon/issues/2237